src/midi/fluid_midi.c

```diff
@@ -1608,7 +1609,9 @@ fluid_track_send_events(fluid_track_t *track,
 
         if(event->type == MIDI_SET_TEMPO && player != NULL)
         {
-            fluid_player_set_midi_tempo(player, event->param1);
+            /* memorize the tempo change value comming from the MIDI file */
```

Comment:
typo: comming -> coming
(can also be found in other places, I only report this one)